packages/@react-spectrum/tooltip/stories/Tooltip.stories.tsx

```diff
@@ -53,32 +53,23 @@ storiesOf('Tooltip', module)
     () => render(longMarkup)
   )
   .add(
-    'triggered by click, placement: left',
-    () => renderWithTrigger('This is a tooltip.', {placement: 'start', type: 'click'})
+    'triggered, placement: left',
+    () => renderWithTrigger('This is a tooltip.', {placement: 'start'})
```

Comment:
do we want separate stories for Tooltip and TooltipTrgger?

Reply:
Yes I think that would a good idea. Following menu / menu trigger & dialog / dialog trigger pattern.